Cargo.toml

```diff
@@ -1,38 +1,5 @@
-[package]
-name = "unic"
-version = "0.4.0"
-authors = ["The UNIC Project Developers"]
-homepage = "https://github.com/behnam/rust-unic/"
-repository = "https://github.com/behnam/rust-unic/"
-license = "MIT/Apache-2.0"
-keywords = ["text", "unicode"]
-description = "UNIC - Unicode and Internationalization Crates"
-categories = ["parsing", "rendering", "encoding", "development-tools"]
-readme = "README.md"
-
-exclude = [
-    "data/**",
-    "tools/**",
-    ".cargo/**",
-    # Only ignore conformance tests;
-    # Those are the ones that rely on data files
-    "tests/*conformance*.rs",
-]
-
 [workspace]
-
-[features]
-default = []
-unstable = []  # Rust nightly features
-bench_it = ["unic-bidi/bench_it"]
-serde = ["unic-bidi/serde"]
+members = [ "unic/" ]
 
 [dependencies]
-unic-ucd = { path = "components/ucd/", version = "0.4.0" }
-unic-bidi = { path = "components/bidi/", version = "0.4.0" }
-unic-idna = { path = "components/idna/", version = "0.4.0" }
-unic-normal = { path = "components/normal/", version = "0.4.0" }
-
-[badges]
-travis-ci = { repository = "behnam/rust-unic", branch = "master" }
-appveyor = { repository = "behnam/rust-unic", branch = "master", service = "github" }
+unic = { path = "unic", version = "0.4.0" }
```

Comment:
If this is a virtual manifest, should it have the `[dependencies]` key? It discovers all of the packages correctly with `[workspace]`. Does this do something for just running `cargo build`?